bin/requirements.txt

```diff
@@ -9,7 +9,7 @@ latexcodec>=1.0.7
 lxml>=4.2.0
 nltk
 pre-commit
-pybtex==0.22.2
+pybtex~=0.24.0
```

Comment:
The upgrade of this package is breaking our codebase when ingesting proceedings such as CCL. In CCL proceeding bib files, they include two author lines, one in `EN` and one in `ZH`. See example below:
<img width="912" alt="Screenshot 2023-08-11 at 3 07 19 PM" src="https://github.com/acl-org/acl-anthology/assets/126604033/31fb428d-8d67-4330-9417-ed08eb329166">

The newer version of pybtex can not handle duplicate person fields (throws an error). See their version history: https://docs.pybtex.org/history.html 

@mbollmann @mjpost I'm changing the version back to 0.22.2. Please let me know any objections.

Reply:
There shouldn't be two author lines like that. I don't think it's valid BibTeX. If they want to add an alternate name representation, they should use ACLPUB2 instead, which uses YAML files instead of BibTeX as the delivery format.

Reply:
BibTeX is quite lenient; it throws a warning and just ignores the second author line. But I agree that this is not good practice. Even renaming the second author field to something non-standard would be preferable to having a duplicate field, IMO.

Reply:
Also, going back to 0.22.2 would mean that we can't build the site locally with Python 3.10+, and my system doesn't actually come with a Python version older than that.

Reply:
Thanks for the feedback.

We have been allowing CCL to submit two line authors in their bib, and we have been using `bin/author_names_to_variants.py` to generate the Chinese name variant, so the final author looks something like this in the xml: 

```
<author><first>Jiayu</first><last>Ding</last><variant script="hani"><first>丁,</first><last>佳玙</last></variant</author>
```

There is an CCL ingestion going on. For the current ingestion,  I'm handing it the old way (which means we downgrade the pybtex version). 

Moving forward, I can let them know that they need to use ACLPUB2. Does this sound good?

Reply:
Yep, this sounds good. Maybe just downgrade in your local environment? And ACLPUB2 for the future.